### PR TITLE
Update selecting your java version page

### PR DESCRIPTION
**Pull request by leonard-IMBERT**
Update the documentation about selecting a version for java with the new system of environement variable

### CODE REVIEW

File: runtimes/java/select-java-version.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ tags:
 
 ## Selecting your java version
 
-Simply create a file **clevercloud/java_version** containing just the java version you want inside.
+Simply set the environement varaiable **JAVA_VERSION** to the version you want
```
Comment (BlackYoup) [inline runtimes/java/select-java-version.md]: environement => environment
varaiable => variable

File: runtimes/java/select-java-version.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ tags:
...
 ## Selecting your java version
 
-Simply create a file **clevercloud/java_version** containing just the java version you want inside.
+Simply set the environment variable **JAVA_VERSION** to the version you want
 
 Accepted values are `6`, `7` and `8`
+
+## Retrocompatibility with the old configuration system
+
+If you have set your java version with the old system (wich consist by set the version in a file **clevercloud/java_version**), don't worry, we guarantee retrocompatibility with this configuration
```
Comment (BlackYoup) [inline runtimes/java/select-java-version.md]: Something like:

The configuration of the JAVA_VERSION through the **clevercloud/java_version** file still works. This is totally backward compatible.

?

Comment (leonard-IMBERT) [inline runtimes/java/select-java-version.md]: Seems better indeed

Comment (leonard-IMBERT) [inline runtimes/java/select-java-version.md]: Do you think the `Retrocompatibility with the old configuration system` need to be change ?

Comment (BlackYoup) [inline runtimes/java/select-java-version.md]: Maybe you should get ride of the title and just add:

"As for the applications created before <month/day/year the update happened>, their JAVA_VERSION configuration can still be made through the **clevercloud/java_version** file. This is backward compatible."

And I think you should add a note about contacting the support if the user wants to migrate from the old way to the new, giving its appID.

**Pull request merged**